src/oss/langchain/rag.mdx

```diff
@@ -787,29 +799,31 @@ agent = create_agent(llm, tools=[], prompt=prompt_with_context)
 :::
 :::js
 ```typescript
-import { createAgent } from "langchain";
+import { createAgent, dynamicSystemPromptMiddleware } from "langchain";
```

Comment:
@christian-bromann are we removing this prebuilt middleware in favor of modifyRequest?

Reply:
no, we decided to keep `dynamicSystemPromptMiddleware` in JS